tripleo-heat-templates-diff/apply_patch.py

```diff
@@ -40,44 +41,69 @@
 PRE_VERSION_3_DIFF = "diff_OSPD10_5.3.3-1"
 POST_VERSION_3_DIFF = "diff_OSPD10_5.3.10-17"
 
-if len(sys.argv) != 1:
-    print "Usage: %s" % sys.argv[0]
-    sys.exit(1)
 
 def version_compare((e1, v1, r1), (e2, v2, r2)):
```

Comment:
Can we add documentation of this method ; what is input and what is output

Reply:
Done